### DISCUSSION TITLE
How much Amiga can the Pi 400 handle?

### DISCUSSION THREAD

**Discussion by FredMSloniker** [BlitterStudio/amiberry#899] [Q&A] [answered]
I'm considering getting a Pi 400 to use as a dedicated Amiga emulation platform and free up resources on my actual compy. I'm curious, though: how powerful an Amiga can I emulate with Amiberry without difficulty? I'm asking on the Pi forums as well, but will I be able to, for instance, emulate an A1200 with an accelerator and FPU and a bunch of RAM at 100% accuracy?

**Comment by solskogen** [comment, marked as answer]
Accelerator and accuracy doesn't match. But yes, it will be pretty awesome. A1200 is no problem. At all.

**Reply by FredMSloniker** [reply]
Well, I mean the kind of accelerators you can get for the A1200. Like the trapdoor 68030s with FPUs and RAM. I don't mean emulator turbo.

**Reply by solskogen** [reply]
You can have a A1200 with a 020, 030 or 040, at different speeds. Same with FPU. 6881 or 6882 on 020 and 030, while the 040 uses the one on the CPU. It supports up to 1GB of memory on z3, but due to some bugs/features of the AmigaOS you should max have 512MB.
